README.md

```diff
@@ -1,52 +1,67 @@
-# Git SCM plugin
+# Git SCM plugin for Jenkins
 
-Git software configuration management for Jenkins
+![Plugin Version](https://img.shields.io/jenkins/plugin/v/git.svg?label=version) [![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins/git-plugin/master)](https://ci.jenkins.io/job/Plugins/job/gitplugin/job/master/)
 
-* see [Jenkins wiki](https://plugins.jenkins.io/git) for feature descriptions
-* use [JIRA](https://issues.jenkins-ci.org) to report issues / feature requests
+Git software configuration management for Jenkins.
 
-## Master Branch
+* see [Jenkins Wiki](https://plugins.jenkins.io/git) for feature descriptions
+* use [Jenkins JIRA](https://issues.jenkins-ci.org) to report issues or request features
 
-The master branch is the primary development branch.
+## Requirements
 
-Branch names using the pattern 'stable-x.y' are development branches
-for changes from a base release 'x.y'.  For example, stable-3.9 is the
-branch used to release fixes based on git plugin 3.9 while master branch
-development is preparing for the 4.0.0 release.
+* Jenkins `2.121.1` or newer
 
-## Contributing to the Plugin
+## Development
+
+### Branches
+
+The `master` branch is the primary development branch.
+
+Branches using name pattern `stable-x.y` are development branches
+for changes from a base release `x.y`. For example `stable-3.9` is the
+branch used to release fixes based on git plugin `3.9`.
```

Comment:
Branch naming pattern is about to change so that we have a single stable-3.x branch instead of using stable-3.n.  That avoids having to create a branch for each minor release.  Coming soon

Reply:
Should we leave it like this for now?

Reply:
The stable-3.x branch has been created and the stable-3.9 branch has been deleted.  Would you be willing to update this pull request to describe the new behavior?

Reply:
OK. Will do.